presentation/src/main/java/com/yapp/growth/presentation/ui/main/detailPlan/DetailPlanScreen.kt

```diff
@@ -0,0 +1,139 @@
+package com.yapp.growth.presentation.ui.main.detailPlan
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Scaffold
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.yapp.growth.presentation.R
+import com.yapp.growth.presentation.component.PlanzExitAppBar
+import com.yapp.growth.presentation.theme.*
+import com.yapp.growth.presentation.ui.main.detailPlan.DetailPlanContract.*
+import kotlinx.coroutines.flow.collect
+
+@Composable
+fun DetailPlanScreen(
+    viewModel: DetailPlanViewModel = hiltViewModel(),
+    exitDetailPlanScreen: () -> Unit,
+) {
+
+    LaunchedEffect(key1 = viewModel.effect) {
+        viewModel.effect.collect { effect ->
+            when (effect) {
+                is DetailPlanSideEffect.ExitDetailPlanScreen -> {
+                    exitDetailPlanScreen()
+                }
+            }
+        }
+    }
+
+    Scaffold(
+        topBar = {
+            PlanzExitAppBar(
+                title = stringResource(id = R.string.detail_plan_app_bar_text),
+                onExitClick = { viewModel.setEvent(DetailPlanEvent.OnClickExitButton) }
+            )
+        },
+        backgroundColor = BackgroundColor1,
+    ) {
+
+        Image(
+            modifier = Modifier
+                .fillMaxSize(),
+            imageVector = ImageVector.vectorResource(R.drawable.ic_detail_plan_character),
+            contentDescription = null
+        )
+
+        Spacer(modifier = Modifier.height(16.dp))
+        Column(
+            modifier = Modifier.padding(horizontal = 21.dp)
```

Comment:
```suggestion
        Column(
            modifier = Modifier
                .padding(top = 16.dp)
                .padding(horizontal = 21.dp)
```

`Spacer` 대신 그냥 패딩 넣어줘도 괜찮을 것 같아요!

Reply:
근데 얘도 뭐가 더 좋은 건지는 모르겠네요... ㅜ